src/components/navigation/HaukiNavigation.tsx

```diff
@@ -51,7 +54,7 @@ export default function HaukiNavigation(): JSX.Element {
           <Navigation.Item
             as="button"
             type="button"
-            onClick={(): void => setAuthenticated(false)}
+            onClick={(): void => console.log('plaa')}
```

Comment:
Are we confident that hits console.log will not be forgotten here? :D.

Reply:
Nope 😁 
I'll remove those

Reply:
Removed the whole signout-button